### PR TITLE
Package plotly.1.0.0

### PR DESCRIPTION
**Pull request by camlspotter**
### `plotly.1.0.0`
Binding for Plotly Open Source Graphing Library
OCaml wrapper for Plotly Open Source Graphing Library: https://plotly.com/graphing-libraries

---
* Homepage: https://github.com/camlspotter/plotly-ocaml
* Source repo: git+https://github.com/camlspotter/plotly-ocaml.git
* Bug tracker: https://github.com/camlspotter/plotly-ocaml/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/plotly/plotly.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Binding for Plotly Open Source Graphing Library"
+description:
+  "OCaml wrapper for Plotly Open Source Graphing Library: https://plotly.com/graphing-libraries"
+maintainer: ["Jun FURUSE <jun.furuse@dailambda.jp>"]
+authors: ["Jun FURUSE <jun.furuse@dailambda.jp>"]
+license: "MIT"
+homepage: "https://github.com/camlpsotter/plotly-ocaml"
```
Comment (mseri) [inline packages/plotly/plotly.1.0.0/opam:9]:
```suggestion
homepage: "https://github.com/camlspotter/plotly-ocaml"
```

File: packages/plotly/plotly.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+bug-reports: "https://github.com/camlpsotter/plotly-ocaml/issues"
```
Comment (mseri) [inline packages/plotly/plotly.1.0.0/opam:10]:
```suggestion
bug-reports: "https://github.com/camlspotter/plotly-ocaml/issues"
```

File: packages/plotly/plotly.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/camlpsotter/plotly-ocaml.git"
```
Comment (mseri) [inline packages/plotly/plotly.1.0.0/opam:34]:
```suggestion
dev-repo: "git+https://github.com/camlspotter/plotly-ocaml.git"
```

Comment (camlspotter) [reply packages/plotly/plotly.1.0.0/opam:34]: Ouch.  Thank you. I will fix `dune-project` and resubmit.

**Comment by mseri** [comment]
Have you re-tagged your release?
```
OpamSolution.Fetch_fail("https://github.com/camlspotter/plotly-ocaml/archive/refs/tags/v1.0.0.tar.gz (Bad checksum, expected md5=08a2810e2a1102e799ce961047b3ee98)")
```
In that case you need to update the hashes.

Note that the opam file in the release does not need to be corrected: since opam will use the one from opam-repository anyway, it is enough to fix it upstream without re-tagging

**Comment by mseri** [comment]
The remaining lower bound issue makes no sense. Opam is installing `pyml 2016` where there is a lower bound in the package asking for `>= 2022`

**Comment by mseri** [comment]
Ah no, I take it back, the issue is in github linkng the wrong results, sorry

**Comment by mseri** [comment]
The remaining issue is independent of this and I am looking into it in a separate PR. Thanks a lot. Feel free to announce the release on discuss.ocaml.org. We have a community category and an announcement tag for it

**Comment by camlspotter** [comment]
Thanks!

**Pull request merged**